docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -119,10 +165,10 @@ has no suffix (and is what the end user adds to their project).
 NOTE: The above `maven-compiler-plugin` configuration requires version 3.5+.
 
 [WARNING]
-====
+=====
```

Comment:
I don't think that was necessary or even if Asciidoc supports it correctly.

Reply:
It allows multiline admonitions or admonitions containing other blocks. It also allows adding title. 
But, yes, in this case, no need.

Reply:
I did `s/^===/====/`, sorry :D